lib/editor/tunnel.js

```diff
@@ -169,10 +169,20 @@ class EditorTunnel {
                     // ↓ useful for debugging but very noisy
                     // console.log('Making a request to:', fullUrl, 'x-access-token:', request.method, reqHeaders['x-access-token'])
                     // debug(`proxy [${request.method}] ${fullUrl}`)
+                    // debug(`body ${request.body}`)
+                    let body = request.body
+                    try {
+                        const temp = JSON.parse(body)
+                        if (typeof temp === 'string') {
+                            body = temp
+                        }
+                    } catch (err) {
+                        // ignore
+                    }
```

Comment:
```suggestion
                    // debug(`body ${request.body}`)
                    let body = request.body
                    if (body && body[0] === '"') {
                        try {
                            body = JSON.parse(body)
                        } catch (err) {
                            // ignore
                        }
                    }
```

This feels an expensive way to only parse strings. Suggest adding a check to see if it looks like a JSON encoded String before attempting the parse.